framework/doc/content/getting_started/installation/manual_installation_llvm.md

```diff
@@ -39,8 +39,6 @@ export LD_LIBRARY_PATH=$PACKAGES_DIR/llvm-__LLVM__/lib:$LD_LIBRARY_PATH
 
 !include manual_miniconda.md
 
-!include manual_ownership.md
```

Comment:
do we still need this file? It seems to only be referenced here and the gcc one.